msg/tools/px_generate_uorb_topic_files.py

```diff
@@ -118,6 +119,21 @@ def generate_output_from_file(format_idx, filename, outputdir, templatedir, incl
         msg_context = genmsg.msg_loader.MsgContext.create_default()
         full_type_name = genmsg.gentools.compute_full_type_name(PACKAGE, os.path.basename(filename))
         spec = genmsg.msg_loader.load_msg_from_file(msg_context, filename, full_type_name)
+        field_name_and_type = {}
+        for field in spec.parsed_fields():
+                field_name_and_type.update({field.name:field.type})
+        # assert if the timestamp field exists
+        try:
```

Comment:
You can change this to one line like:
`if 'timestamp' not in field_name_and_type:`

Reply:
I actually do think it is a better practice to use a try catch approach for this kind of assertions, rather than using if/else.

Reply:
Generally yes, but not if you catch it with the next line.
Not a big deal to me however.